table/table.go

```diff
@@ -189,111 +184,49 @@ func (t *Table) readNoFail(off int, sz int) []byte {
 	return res
 }
 
-func (t *Table) readIndex() error {
+func (t *Table) readIndex() {
 	readPos := t.tableSize
 
 	// Read bloom filter.
 	readPos -= 4
 	buf := t.readNoFail(readPos, 4)
-	bloomLen := int(binary.BigEndian.Uint32(buf))
+	bloomLen := int(binary.LittleEndian.Uint32(buf))
```

Comment:
It's very easy to make mistake when use LittleEndian, it's better to extract a function.